provider/views.py

```diff
@@ -580,9 +580,10 @@ def access_token_response(self, access_token, nonce=''):
         Returns a successful response after creating the access token
         as defined in :rfc:`5.1`.
         """
+        import simplejson as simple_json
         response_data = self.access_token_response_data(access_token, nonce=nonce)
         return HttpResponse(
-            json.dumps(response_data), content_type='application/json'
+            simple_json.dumps(response_data), content_type='application/json'
```

Comment:
Is response_data bytes in python 3 but string in python2?  In that case, can we wrap it in `six.text_type` to get a valid string in both cases?  If we do that we don't need to add a new library to the dependencies.

Reply:
ok i will check that out. have we ran modernize on this repo.

Reply:
It looks like this repo is a fork of a 3rd party app @Ayub-Khan, have you looked at what the differences are between upstream an us?